app/Http/Controllers/Admin/StandAdminController.php

```diff
@@ -149,6 +149,37 @@ public function deleteStand(Airfield $airfield, Stand $stand) : JsonResponse
         return response()->json([], 204);
     }
 
+    /**
+     * Get a list of terminals for an Airfield if configured.
+     *
+     * @param Airfield $airfield
+     * @return JsonResponse
+     */
+    public function getTerminals(Airfield $airfield) : JsonResponse
+    {
+        $terminals = $airfield->load('terminals')->terminals;
+
+        if ($terminals->count() < 1) {
```

Comment:
Would it be better to reserve 404 for just "no airfield can be found" and return an empty JSON object for no terminals?

Reply:
My thinking, at least from a front-end perspective, was that you'd try this endpoint first to see if any terminals existed, and then displayed a terminal-centric view. I guess either works. I chose 404 because it made more semantic sense